### PR TITLE
Added a few stuff

### PR DESCRIPTION
**Pull request by wallace11**
- Material Design for options & popup

- Reverted to old good css (from scss)

- Much more useful popup, with the ability to paste in links and view download history

- Add support for unrestricting folders (tested against uploaded.net) 

- Upgraded notifications including:

   1. Progress notification
   2. More informative error messages
   3. Copying unrestricted link 

-  Fixing redirection to RD torrents page after adding a magnet

- jQuery 3.1

- Commented getTraffic for now...

Apologies for the initial bunch of commits, not used to working on Github...

### CODE REVIEW

File: src/js/background.js [hunk 1]
```diff
@@ -176,33 +180,59 @@ function RealDebrid(warningPercentage, warningDays, splittingSize, torrentHost)
     this.selectionHandler = function(selection) {
         var urls = selection.split(" ");
         var regex = new RegExp(/((([A-Za-z]{3,9}:(?:\/\/)?)(?:[-;:&=\+\$,\w]+@)?[A-Za-z0-9.-]+|(?:www.|[-;:&=\+\$,\w]+@)[A-Za-z0-9.-]+)((?:\/[\+~%\/.\w-_]*)?\??(?:[-\+=&;%@.\w_]*)#?(?:[\w]*))?)/);
+        var invalid;
         $.each(urls, function(index, url) {
             if (url.match(regex)) {
                 that.urlHandler(url);
+            } else {
+                invalid = 1;
```
Comment (JDevlieghere) [inline src/js/background.js]: Why not `invalid = true`?

File: src/js/background.js [hunk 2]
```diff
@@ -348,15 +414,25 @@ function Notifier() {
...
-            if (onClicked) {
-                that.callbacks[notificationId] = onClicked;
-            }
-        });
+
+        if (progress == 100) {
+            options.buttons = [{ title: "Copy URL To Clipboard", iconUrl: "/icons/copy_18dp_2x.png" }];
+        }
+        if (progress > 0) {
+            chrome.notifications.update("prog_" + id, options, function(){
```
Comment (JDevlieghere) [inline src/js/background.js]: Make "prog_" an ALL_CAPS constant

**Comment by JDevlieghere** [comment]
Hi Wallace,

Thank you for the Pull-Request! I quickly glanced over the code at it seems like you made a lot of improvements, which is of course greatly appreciated. I added a few minor comments on things that caught my eye but nothing that would prevent me from merging it in. I wonder why you got rid of the SCSS though? 

I'll test drive it for a day and have @stefanpante look it over before I pull in your changes.

Thanks again for the contribution! 

Cheers,
Jonas

File: src/js/background.js (hunk 1 above)
Comment (wallace11) [inline src/js/background.js]: You're right. I'll fix that asap!

File: src/js/background.js (hunk 2 above)
Comment (wallace11) [inline src/js/background.js]: Not exactly sure what you mean by that. Should we convert all the prefixes for notifications to caps?

**Comment by wallace11** [comment]
Hi Jonas,

Thank you very much for the quick reply. I appreciate any comments on the changes I've made. 
There are a few places in which I felt that a different method should be used (including in my own changes), and I added a "TODO" comment in some of them. 

As for the SCSS > CSS transition - I used Material Design Lite (MDL) since I figured it was much lighter and suited for an extension than other MD implementations such as Polymer (which, by the way, uses a huge amount of memory). I thought of MDL as a complete MD solution, but later found it out to be a rather incomplete and inconsistent one. A V2 is apparently on its way (maybe with the release of Android N?), and when it's out I plan to review the design part again. Anyhow,  it took me a while to tame this beast, and working directly with CSS was much easier. 
Are there any benefits for using SCSS over CSS in such a small project?

BTW, I edited the third point on my pull-request, seems like half a sentence was missing there ;)

Cheers,
Wallace

File: src/js/background.js (hunk 2 above)
Comment (JDevlieghere) [inline src/js/background.js]: I just meant that you could make the prefixes constants like `var NOTIFICATION_ID_PREFIX = 'id_'`. It's less error-prone and more maintainable.

**Comment by wallace11** [comment]
Alright, so I implemented your suggestions, fixed a very embarrassing miss on my end, where I didn't revert changes I made for testing, and did some more cleanup. 
Should I create a new pull request for the new commit...?

**Comment by JDevlieghere** [comment]
Awesome! You should be able to just push it to your repository and the PR should be updated automatically. (More info on [SO](http://stackoverflow.com/questions/9790448/how-to-update-a-pull-request))

File: src/js/background.js [hunk 3]
```diff
@@ -176,33 +180,59 @@ function RealDebrid(warningPercentage, warningDays, splittingSize, torrentHost)
...
-                        }, function() {});
+            that.handleMagnet(url, function(result) {
+                if (result.uri) {
+                    chrome.tabs.create({
+                        url: "https://real-debrid.com/torrents"
                     });
                 } else {
-                    nf.basic(result.message, url);
+                    nf.error("Error adding magnet");
                 }
```
Comment (stefanpante) [inline src/js/background.js]: for the sake of consistency, every string should be declared the same way.  with single quotes or double quotes. not both

Comment (wallace11) [inline src/js/background.js]: I just checked and all notifications use double quotes. 
Reverting all to single quotes means that any word like "doesn't" will become a little bit of a pain, wouldn't it?

Comment (stefanpante) [inline src/js/background.js]: all strings means also your urls, etc. Maybe we should add an .eslintrc file in the future to enforce style.

Comment (wallace11) [inline src/js/background.js]: Gotcha. Will convert all to double quotes.

**Pull request merged**